runtime/src/lib.rs

```diff
@@ -13,3 +13,4 @@ pub use nexus_rt_macros::{main, profile};
 
 mod ecalls;
 pub use ecalls::*;
+pub use postcard::*;
```

Comment:
It would probably be preferable to just do `pub use postcard`, and have users access through `nexus_rt::postcard` to be clear what's going on here.